ci/base-images/README.md

```diff
@@ -14,18 +14,22 @@ Below table summarizes all available container image versions. These images incl
+| Java     | 21                           | ghcr.io/cyclonedx/cdxgen-temurin-java21:v11                                                    | Java 21 version SDK.                                                                                                                     |
+| Java     | 24                           | ghcr.io/cyclonedx/cdxgen-temurin-java24:v11                                                    | Java 24 version SDK.                                                                                                                     |                     
 | Dotnet   | .Net Framework 4.6 - 4.8     | ghcr.io/cyclonedx/cdxgen-debian-dotnet6:v11                                                    | .Net Framework                                                                                                                            |
 | Dotnet   | .Net Core 2.1, 3.1, .Net 5.0 | ghcr.io/cyclonedx/cdxgen-debian-dotnet6:v11                                                    | Invoke with --platform=linux/amd64 for better compatibility.                                                                              |
 | Dotnet   | .Net 6                       | ghcr.io/cyclonedx/cdxgen-debian-dotnet6:v11                                                    | .Net 6                                                                                                                                    |
 | Dotnet   | .Net 7                       | ghcr.io/cyclonedx/cdxgen-dotnet7:v11 (amd64 only)                                              | .Net 7                                                                                                                                    |
 | Dotnet   | .Net 8                       | ghcr.io/cyclonedx/cdxgen-debian-dotnet8:v11, ghcr.io/cyclonedx/cdxgen-dotnet8:v11 (amd64 only) | .Net 8                                                                                                                                    |
-| Dotnet   | .Net 9                       | ghcr.io/cyclonedx/cdxgen-debian-dotnet9:v11, ghcr.io/cyclonedx/cdxgen-dotnet9:v11 (amd64 only) | .Net 9                                                                                                                                    |
+| Dotnet   | .Net 9                       | ghcr.io/cyclonedx/cdxgen-debian-dotnet9:v11, ghcr.io/cyclonedx/cdxgen-dotnet9:v11 (amd64 only) | .Net 9                                                              |      
+| Dotnet   | .Net 10                      | ghcr.io/cyclonedx/cdxgen-debian-dotnet10:v11, ghcr.io/cyclonedx/cdxgen-dotnet10:v11 (amd64 only) | This is a preview image of .Net 10                                                                |
```

Comment:
Remove `ghcr.io/cyclonedx/cdxgen-dotnet10:v11 (amd64 only)` since we are publishing dotnet10 for debian only (without SLE).